src/scripts/inc/common.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+TMP_DIR="/tmp"
+RUN_USER="alexapi"
+HOME_DIR="/var/lib/AlexaPi"
+
+function run_python {
+    /usr/bin/python2 $@
+}
+
+function run_pip {
```

Comment:
Good point there! Changed to the _which_ stuff :smile: 
I'm keeping the run_ instead of install_, since now we actually use it and I have a hunch that we might use it for other things also.
Thanks!